formulas/wick-base/functions/wick-is-apt-available

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+# Checks whether Apt and Dpkg type applications are running.
+#
+# Examples
+#
+#   wickIsAptAvailable
+#
+# Returns 0 when no Apt or Dpkg type applications are found running.
+# Returns >0 when an Apt or Dpkg type application is found running.
```

Comment:
The plural / non plural I think is correct, the first one checks multiple applications and finds none running at all where as the second statements refers to a single one found running. In theory there could be multiple running but it returns when it finds a single one. I will add the space.